pkg/operator/controllers/clusteroperatoraro/clusteroperatoraro_controller.go

```diff
@@ -156,6 +158,12 @@ func (r *Reconciler) defaultOperator() *configv1.ClusterOperator {
 			Name: clusterOperatorName,
 		},
 		Status: configv1.ClusterOperatorStatus{
+			Versions: []configv1.OperandVersion{
+				{
+					Name:    "operator",
+					Version: fmt.Sprintf("1.0-%s", version.GitCommit),
```

Comment:
Is there a reason we use a prefix, or could we get away with just the commit here?
```suggestion
					Version: version.GitCommit,
```